backend/app/services/rag/pipeline/docling.py

```
@@ -0,0 +1,325 @@
+                    f"{self.docling_url}/convert",
+                    files=files,
+                    params=params,
+                )
+
+                if response.status_code != 200:
+                    error_detail = (
+                        response.text[:500] if response.text else "No details"
+                    )
+                    raise DoclingServiceError(
+                        f"Docling API returned status {response.status_code}: {error_detail}"
+                    )
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)